spec/base_model_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative "spec_helper"
+require "cfa/augeas_parser"
+require "cfa/base_model"
+require "cfa/memory_file"
+
+describe CFA::BaseModel do
+  let(:parser) { CFA::AugeasParser.new("sudoers.lns") }
+  let(:path) { "/etc/sudoers" }
+  subject { CFA::BaseModel.new(parser, path)}
+
+  describe ".default_file_handler" do
+    it "returns object set by .default_file_handler" do
```

Comment:
I know, better idea how to test it? it is even very trivial part of code, so I am not sure if it deserve test. It is basically cattr_accessible just with default value ( rails can do not it, but ruby probably not ).